modules/oe_translation_poetry/config/schema/oe_translation_poetry.schema.yml

```diff
@@ -28,3 +27,44 @@ field.value.poetry_request_id:
     product:
       type: string
       label: Product
+tmgmt.translator.settings.poetry:
+  type: tmgmt.translator_base
+  mapping:
```

Comment:
Should these config values be under the `settings` key of the translator config by the way?